diagnostic.js

```diff
@@ -32,6 +49,23 @@ db.dropDatabase();
 // Jon Arryn
 
 // your code begins here
+db.houses.insert( {name: "House Stark"},
+  {
+    "members" : [ "Ned Stark", "Arya Stark", "Sansa Stark" ]
+  }
+);
+
+db.houses.insert( {name: "House Targaryen"},
+  {
+    "members" : [ "Viserys Targaryen", "Daenerys Targaryen" ]
+  }
+);
+
+db.houses.update( {name: "House Arryn"},
+  {
+    "members" : [ "Jon Arryn" ]
+  }
+);
```

Comment:
Actually, this is not correct. `.insert` adds a document to a collection. What you want is an `update` action, but remember that with `update`, the 1st argument is the thing we are searching by. If you are not using a method like $set or $push in the 2nd argument, that object will simply get replaced by the 2nd argument.